a/apache-nifi/build_info.json

```diff
@@ -20,8 +20,13 @@
         "build_script" : "apache-nifi_1.17.0_ubi_8.5.sh"
     },
 
-    "rel/nifi-*.*.*": {
+    "rel/nifi-1.22.0": {
         "dir":"1.17.0_ubi_8",
```

Comment:
@vishakadesai Is this correct folder name?

Reply:
made the corrections @nikhil-kalbande